connector/spanmetricsconnector/README.md

```diff
@@ -33,6 +33,28 @@ across all spans:
 - `span.kind`
 - `status.code`
 
+
+## Span to Metrics processor to Span to metrics connector
+
+The spanmetrics connector is a port of the [spanmetrics](../../processor/spanmetricsprocessor/README.md) processor, but with multiple improvements
+and breaking changes. It was done to bring the `spanmetrics` connector closer to the OpenTelemetry
+specification and make the component agnostic to exporters logic. The `spanmetrics` processor
+essentially was mixing the OTel with Prometheus conventions by using the OTel data model and
+the Prometheus metric and attributes naming convention.
+
+The following changes were done to the connector component.
+
+Breaking changes:
```

Comment:
We should call out specifically that the number of metrics will increase because we now add service.instance.id

Reply:
done https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/19471/commits/007bc574cf849d3ad15b08ced8e5cf0d6dea8321